glib/src/date_time.rs

```diff
@@ -0,0 +1,17 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
```

Comment:
It seems weird to add a file containing only a test. Can you put it into the `tests` folder instead please?

Reply:
It increases build time when running tests only. Or am I misunderstanding you?

Reply:
Yes, but it seems completely unnecessary :) We might add new manual datetime API there soonish anyway.

Reply:
Also conceptually this is an unit test, not an integration test

Reply:
That triggers strong OCD in me but whatever. Let's go with that.

Reply:
Funny because the other way would do that for me :)